Dockerfile

```diff
@@ -5,22 +5,22 @@ FROM rust:latest as builder
 WORKDIR /usr/src/microbin
 
 # copy sources to /usr/src/microbin on the temporary container
-COPY . .
+COPY Cargo.toml render.yaml .
+COPY templates ./templates
+COPY src ./src
 
 # run release build
 RUN cargo build --release
 
 # create final container using slim version of debian
 FROM debian:buster-slim
 
-# microbin will be in /usr/local/bin/microbin/
-WORKDIR /usr/local/bin
+WORKDIR /app
```

Comment:
Can you please add a comment for this step? Just so when we come back here later we know what it does. :-)